Libs/vtkSegmentationCore/vtkSegmentation.cxx

```diff
@@ -1808,14 +1881,14 @@ std::string vtkSegmentation::AddEmptySegment(std::string segmentId/*=""*/, std::
   }
 
   // Segment ID will be segment name by default
-  segmentId = this->GenerateUniqueSegmentID(segmentId);
   if (!segmentName.empty())
   {
     segment->SetName(segmentName.c_str());
   }
   else
   {
-    segment->SetName(segmentId.c_str());
+    std::string name = this->GenerateUniqueSegmentName("Segment");
```

Comment:
We have to pass in this segment name pattern from outside (from i18n-aware code)